live-build/config/hooks/vm-artifacts/90-linux-migration-artifact.binary

```diff
@@ -120,6 +120,14 @@ cp migration-scripts/* $DEPOT_DIRECTORY
 	# shellcheck disable=SC2016
 	echo "DLPX_VERSION=$(chroot binary dpkg-query \
 		-Wf '${Version}' delphix-virtualization)"
+
+	#
+	# DLPX_PLATFORM is set to the type of platform this migration
+	# image was built for (i.e. internal-qa-esx). It will be used to
+	# check that we're using the correct image type for the engine
+	# being upgraded.
+	#
+	echo "DLPX_PLATFORM=$ARTIFACT_NAME"
```

Comment:
We should probably use `APPLIANCE_PLATFORM` instead of `ARTIFACT_NAME ` here as that's what we are really interested in.
We have this relationship defined in `scripts/run-live-build.sh`:
```
export ARTIFACT_NAME="$APPLIANCE_VARIANT-$APPLIANCE_PLATFORM"
```

Reply:
I also talked to John G and he pointed out that I can get the artifact platform from `/var/lib/delphix-appliance/platform` on the new filesystem. Since I'm already mounting that for other checks executed by `dx_verify`, I think I'll just use that and not use `version.info` after all.